README.md

```diff
@@ -1,2 +1,36 @@
 # hw01
 Repository for homework 1
+
+Text added through RStudio
+
+A brief biography as a part of the homework:
+
+## Zahra Amalia Syarifah
+I was born in Indonesia and grew up there. However, shortly before my high school graduation I moved to the UK. *So I **never** actually completed high school*. I went straight to a college in Southeast England and moved to Scotland for my undergraduate. Now, I am completing my Master in the States.
+
+For the sake of using the list. Here are the list of my favourite carbohydrates:
+* Pasta
+* Potato
+* Bread
+
+Here is an image of Hungarian Goulash (for the sake of the points, but also because it's one of my favourite dishes):
+
+![Image of Hungarian Goulash](http://img.taste.com.au/JCSA2ji6/w720-h480-cfill-q80/taste/2016/11/goulash-soup-75934-1.jpeg)
```

Comment:
Included a picture with the correct code. Awesome! It took me a couple of tries to get the code for this right.